src/bokeh/core/property/descriptors.py

```diff
@@ -164,11 +171,48 @@ def __set__(self, obj: HasProps | None, value: T) -> None:
 
     @property
     def readonly(self) -> bool:
-        return self.property.readonly
+        return self.alias.readonly
 
     def has_unstable_default(self, obj: HasProps) -> bool:
         return obj.lookup(self.aliased_name).has_unstable_default(obj)
 
+    def class_default(self, cls: type[HasProps], *, no_eval: bool = False):
+        return cls.lookup(self.aliased_name).class_default(cls, no_eval=no_eval)
+
+class DeprecatedAliasPropertyDescriptor(AliasPropertyDescriptor[T]):
+    """
+
+    """
+
+    alias: DeprecatedAlias[T]
+
+    def __init__(self, name: str, alias: DeprecatedAlias[T]) -> None:
+        super().__init__(name, alias)
+
+        major, minor, patch = self.alias.since
+        since = f"{major}.{minor}.{patch}"
+        self.__doc__ = f"""\
+This is a backwards compatibility alias for the '{self.aliased_name}' property.
+
+.. note::
+    Property '{self.name}' was deprecated in bokeh {since} and will be removed
+    at some point. Update your code to use '{self.aliased_name}' instead.
```

Comment:
```suggestion
    in the future. Update your code to use {self.aliased_name!r} instead.
```
"in the feature" seems more plain / less idiomatic